modules/extensions-mule-support/src/main/java/org/mule/runtime/module/extension/mule/internal/loader/parser/MuleSdkOperationModelParserSdk.java

```diff
@@ -267,7 +267,8 @@ private Optional<ComponentAst> getOutputElement(String elementName) {
   }
 
   private ComponentAst getBody() {
-    return getSingleChild(operation, "body").get();
+    // TODO: What happens if the operation AST doesn't have a body? Is this code reachable in that case?
```

Comment:
validation will fail in that case. At this point, we assume that the syntax is valid.

Reply:
Changed the TO-DO with a comment explaining why we aren't using isPresent there